packages/ListBox/src/components/Divider/Divider.js

```diff
@@ -18,7 +18,7 @@ const defaultProps = {
 
 export default function Divider(props) {
   return (
-    <li css={dividerCSS} hasChildren={!!props.children}>
+    <li data-pka-anchor="listbox-divider" css={dividerCSS} hasChildren={!!props.children}>
```

Comment:
for sub components we use `.` for the separator

Reply:
ok

Reply:
done